packages/client-api-schema/client-api-schema.api.md

```diff
@@ -259,6 +259,9 @@ export type ErrorCodes = {
         NotYourTurn: 403;
```

Comment:
This file (as well as the duplicate under `/temp/` are autogenerated: I'm not sure why they changed given no changes were otherwise made to `client-api-schema`?

Reply:
I wonder if my local environment is different from that of the last machine that built these files? If you rebuild these files on your machines, are there changes?

Reply:
> If you rebuild these files on your machines, are there changes?

Before this PR, yes, after this PR, no. 

So I think that means (although unrelated to this PR) we needed to check these changes in. Or we could just gitignore them for now, since they are only used as a part of the docs generation pipeline at present. And it's weird that there is a duplicate. 

On the other hand, they are potentially useful canaries in our coal mine, as they highlight when we make changes to the API of one of these packages. So we could think about leveraging that somehow.

Reply:
#2614